CONTRIB-234-392: Translate guidelines (#416)
feature-sliced/documentation: CONTRIBUTING.md

[fork]: https://github.com/feature-sliced/documentation/fork
[actions]: https://github.com/feature-sliced/documentation/actions

> <!-- Other emojis: 👁️ , ✍️ , 🔍 -->

Author: Це md автоформаттер 👀

- 📢 [Share feedback, ask and **discuss** anything][disc]
   > We will be glad to receive any feedback from you!
- 💡 [Notify about bugs, suggest improvements][issues-new]
   > If something specific doesn't work quite well for you or could be better - let us know!
- 💬 Rate & discuss [**issues**][issues]
   > Share your opinion, evaluate the problem identified by the author

> - 🔩 Repeat complex [**issues**][issues]
>    > Some issues are difficult to repeat

Reviewer: ```suggestion
- 🔩 Reproduce complex [**issues**][issues]
   > Some issues are difficult to reproduce
```
Если я правильно понимаю, что имелось ввиду

> - 🛡️ Conduct a review for [**pull requests**][pr]

Reviewer: Кмк, `conduct` имеет больше похожую коннотацию на "организовывать"

Мы у себя обычно говорим `provide`

Reviewer: >1. To direct the course of; manage or control: a police officer who conducts traffic; a scientist who conducts experiments.
>2. To lead or guide: conducted the tourists through the museum.

   > Share your opinion and help us with the processing of other people's proposals

> - ⚒️ Suggest [own **pull-requests**!][pr-new]

Reviewer: ```suggestion
- ⚒️ Suggest your [own **pull-requests**!][pr-new]
```
Думаю так лучше звучит

   > Enhance the project with your own solutions

## Workflow

[self-review-article]: https://blog.beanbaginc.com/2014/12/01/practicing-effective-self-review/

> 1. [Fork][fork] repository

Reviewer: ```suggestion
1. [Fork][fork] the repository
```

2. Make your changes
   - Make sure that **commits follow** the [Conventional Commits specification](https://www.conventionalcommits.org)

>       > All this helps the formation of changelog and familiarization with the history of the project

Reviewer: Как-то это `familiarization` мне звучит сильно сложно)

Может перефразировать, сказав что-то наподобие `and keeping the history clean [and searchable]`?

      >
      > *It would also be useful to indicate in each commit (preferably in the body) the ID of the task in the format `#{ID}' (so that there is linking)*
   - Make sure that **all checks pass**

      ```sh
      $ npm run test
      # > Linting is not broken
      # > The build does not crash
      ```

   - To modify/add markdown tables, it is recommended to use ready-made services (for example [this](https://www.tablesgenerator.com/markdown_tables))

> 3. [Offer your pull-request][pr-new] from *your forked branch* and specify related [issues][issues] (if any)

Reviewer: Не могу сказать, что offer некорректно, но не помню, чтобы слышал его в таком контексте. Я бы использовал просто `open a pull request`

>    - Before creating a PR spend at least one [`self-review`][self-review-article] of your changes, to save the reviewer's time

Reviewer: ```suggestion
   - Before creating a PR, do[/have] at least one [`self-review`][self-review-article] of your changes, to save the reviewers' time
```

>    - Also make sure that the description of your PR describes the problem being solved as clearly as possible (take care of the reviewer)

Reviewer: Можно убрать тавтологию как-то так
```suggestion
   - Also make sure that you describe the problem being solved as clearly as possible (take care of the reviewer) in your PR
```

      > The more details you provide in the description, the better
      >

>       > *You can get acquainted with [what pull-request was done before you][pr]*

Reviewer: ```suggestion
      > *You can get acquainted with [what pull-requests were done before you][pr]*
```

>    - If everything is fine, you can ping someone from the core maintainers to speed up

Reviewer: ```suggestion
   - If everything is fine, you can ping someone from the core maintainers to speed things up
```

без какого-тосуществительного внутри звучит как "чтобы ускориться"
